src/main/day1/humanInterface.kt

```
@@ -0,0 +1,10 @@
+package my.demo
+
+
+fun doSomething(){
+    println("hello world")
+}
+
+class Something{
+}
```

detekt (1.23.7)

[warning] The class or object Something is empty.

(detekt.empty-blocks.EmptyClassBlock)